### PR TITLE
change to lowercase

### PR DESCRIPTION
**Pull request by iregina**

### CODE REVIEW

File: pages/en/using-mina/install-a-wallet.mdx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+export default Page({ title: "Install a Wallet" });
+
+# Install a wallet
+Learn how to get a wallet for storing and sending MINA
+
+The Mina community has created a variety of different wallets.
+
+You can send and receive MINA from all of these wallets, but currently only Auro Wallet supports zkApps interactions.
+
+- <a href="https://www.aurowallet.com">Auro Wallet</a> - Chrome, App Store, Firefox, Google Play
```
Comment (jasongitmail) [inline pages/en/using-mina/install-a-wallet.mdx:12]: App Store > iOS
Google Play > Android

File: pages/en/using-mina/install-a-wallet.mdx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+- <a href="https://clor.io">Clorio Wallet</a> - Windows, OSX, Linux, and web browser
+- <a href="https://www.staking-power.com">Staking Power</a> - Android and iOS
+- <a href="https://www.ledger.com/mina-wallet">Ledger Hardware Wallet</a> - See Mina docs <a href="https://docs.minaprotocol.com/en/node-operators/ledger-app-mina">here</a>.
+
+### Wallet for Node Operators
+
+The Mina CLI Wallet is only intended for node operators. We do **not** recommend this Wallet for the average user.
```
Comment (jasongitmail) [inline pages/en/using-mina/install-a-wallet.mdx:19]: `recommend this Wallet for` > `recommend this wallet for` (lowercase wallet)

File: pages/en/using-mina/install-a-wallet.mdx [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+# Install a wallet
+Learn how to get a wallet for storing and sending MINA
+
+The Mina community has created a variety of different wallets.
+
+You can send and receive MINA from all of these wallets, but currently only Auro Wallet supports zkApps interactions.
+
+- <a href="https://www.aurowallet.com">Auro Wallet</a> - Chrome, App Store, Firefox, Google Play
+- <a href="https://clor.io">Clorio Wallet</a> - Windows, OSX, Linux, and web browser
```
Comment (jasongitmail) [inline pages/en/using-mina/install-a-wallet.mdx:13]: `OSX` > `Mac OS`

File: pages/en/participate/grantoverview.mdx [hunk 4]
```diff
@@ -10,13 +10,15 @@ Apply for grants and programs in the Mina ecosystem
...
+zkApps Builders Program (ZBP) is a 12-week, online program to support developers who are interested in building zkApp-related projects.
 
-The goal of the program is to provide direct access to the O(1) Labs team to support developers as they build their zkApp project and for O(1) Labs to gather feedback for improving <DocLink copy="SnarkyJS" url="/zkapps/snarkyjs-reference" /> and <DocLink copy="zkApps" url="/zkapps" />. You can read more about the projects that came out of the [Winter 2022 Cohort here](https://minaprotocol.com/blog/zkapps-builders-program-spring-2022-wrapup).
+The goal of the program is to provide direct access to the O(1) Labs team to support developers as they build their zkApp project and to gather feedback for improving <DocLink copy="SnarkyJS" url="/zkapps/snarkyjs-reference" /> and <DocLink copy="zkApps" url="/zkapps" />. You can read more about the projects that came out of the [Winter 2022 Cohort here](https://minaprotocol.com/blog/zkapps-builders-program-spring-2022-wrapup).
 
-To sign up for the 2022 Summer zkApps Builders Program, please [apply here](https://share.hsforms.com/1KUeCPxJzQC6PIZ-LLdZe1g4xuul).
+Fall 2022 Applications are now closed.
```
Comment (jasongitmail) [inline pages/en/participate/grantoverview.mdx]: Applications > applications

**Review by jasongitmail — APPROVED** [review]

File: pages/en/using-mina/install-a-wallet.mdx (hunk 1 above)
Comment (iregina) [reply pages/en/using-mina/install-a-wallet.mdx:12]: Done

**Pull request merged**